### PR TITLE
TRIVIAL: Add github action to trigger sonarcloud scan

### PR DESCRIPTION
**Pull request by tomaso**
The action runs for each component separately => updates
four sonarcloud projects.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-fdw/sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+sonar.projectKey=gooddata_python-sdk_gooddata-fdw
```
Comment (pcerny) [inline gooddata-fdw/sonar-project.properties:1]: Add also keys:
- `sonar.projectName` - https://docs.sonarcloud.io/advanced-setup/analysis-parameters/
- `sonar.python.version=3.7, 3.8, 3.9, 3.10`
- `sonar.sources` - https://docs.sonarcloud.io/advanced-setup/analysis-scope/
- `sonar.tests` - https://docs.sonarcloud.io/advanced-setup/analysis-scope/

**Review by pcerny — CHANGES_REQUESTED** [review]

File: .github/workflows/sonarcloud.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: sonarcloud
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+  workflow_dispatch:
```
[the remark this answers is not in the document]
Comment (tomaso) [reply .github/workflows/sonarcloud.yaml:8]: Actually no. I will remove it.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-fdw/sonar-project.properties (hunk 1 above)
Comment (tomaso) [reply gooddata-fdw/sonar-project.properties:1]: Added to other projects too.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Pull request closed without merging**